### PR TITLE
Lint OpenAPI Specification

### PR DESCRIPTION
**Pull request by Richienb**
https://github.com/lichess-org/api/pull/129#issuecomment-885055838

### CODE REVIEW

**Comment by Richienb** [comment]
I've finished setting up the linter so all that's left is to go though the lint errors and fix them.

**Comment by ornicar** [comment]
is this ready to go?

**Comment by Richienb** [comment]
Some of the lint errors mention the lack of an entire schema that is supposed to exist. This will take a bit of effort to add.

File: doc/specs/lichess-api.yaml [hunk 1]
```diff
@@ -7400,3 +7383,4 @@ components:
             "msg:write": Send private messages to other players
             "board:play": Play with the Board API
             "bot:play": Play with the Bot API. Only for [Bot accounts](#operation/botAccountUpgrade)
+            "web:mod": Create challenges between players
```
Comment (niklasf) [inline doc/specs/lichess-api.yaml:7386]: The official description of the `web:mod` scope is:

> Use moderator tools (within the bounds of your permissions)

**Comment by Richienb** [comment]
@ornicar Fixed all immediate linting issues so this PR is ready to merge.

It would be a good idea to later chip away at the missing schemas to complete the openapi specification.

**Pull request merged**